src/ActionList2/Group.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import Box from '../Box'
+import {SxProp} from '../sx'
+import {Header, HeaderProps} from './Header'
+import {ListProps} from './List'
+
+export type GroupProps = HeaderProps &
+  SxProp & {
+    selectionVariant?: ListProps['selectionVariant'] | false
+  }
+
+type ContextProps = Pick<GroupProps, 'selectionVariant'>
+export const GroupContext = React.createContext<ContextProps>({})
+
+export const Group: React.FC<GroupProps> = ({title, variant, auxiliaryText, selectionVariant, sx = {}, ...props}) => {
+  return (
+    <Box
+      as="li"
```

Comment:
Looks like Typescript isn't happy because of the `as="li"` 🤔

Reply:
Yeah, any idea why? :( 

cc @colebemis @jfuchs?

Reply:
I wish I had some helpful guidance, but I'm stumped here too :/